docs/howto/operate/new-hub/add-aws-hub.md

```diff
@@ -14,123 +14,112 @@ discussions outlined in [#431](https://github.com/2i2c-org/pilot-hubs/issues/431
 
 ## Pre-requisites
 
-1. Follow the instructions outlined in
-[Set up and use the the deployment scripts locally](./manual-deploy.md) to set up the
-local environment and prepare `sops` to encrypt and decrypt files.
+1. Follow the instructions outlined in [Set up and use the the deployment scripts locally](../manual-deploy.md) to set up the local environment and prepare `sops` to encrypt and decrypt files.
```

Comment:
a lot of the text in this section wasn't properly formatted markdown, so the numbered lists etc weren't working. Most of the changes here are just adding in the proper whitespace so that this works